lib/components/Attachments/constants.js

```diff
@@ -0,0 +1,28 @@
+const MAX_FILE_SIZE = 100 * 1024 * 1024; // 100 MB
+
+export const ALLOWED_FILE_TYPES = [
+  ".jpg",
+  ".jpeg",
+  ".png",
+  ".gif",
+  ".pdf",
+  ".csv",
+  ".xlsx",
+];
```

Comment:
Let's not provide allowed fle types for attachments. There are a large number file extensions that needs to be supported. We'll add it once we have a good idea about the same. Please make the necessary changes in the `useUppyUploader` hook so that this logic is reflected.